azurerm/internal/services/sentinel/sentinel_alert_rule_scheduled_resource.go

```diff
@@ -73,6 +73,24 @@ func resourceSentinelAlertRuleScheduled() *schema.Resource {
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
 
+			"event_grouping_setting": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"aggregation_kind": {
```

Comment:
given there is only one property here? maybe we should pull this out and call it
```suggestion
						"Event_grouping_aggregation_methold": {
```

(kind isn't great, methold is more descriptive)

Reply:
I am not sure whether api would introduce other types in the future. If I put this out, I still need to update the property schema like this and it would become breaking change. Does it make sense?

Reply:
It's "method" not "methold", right?

Reply:
Yes method, and its up to you, honestly its not a big deal to deprecate a string property if they add one

Reply:
OK. Thanks. I prefer to keep it.